cumulusci/core/sfdx.py

```diff
@@ -35,19 +35,20 @@ def sfdx(
 
     Returns a `sarge` Command instance with returncode, stdout, stderr
     """
-    command = f"sfdx {command}"
+    sfdx_cmd = f"sfdx {command}"
     if args is not None:
         for arg in args:
-            command += " " + shell_quote(arg)
+            sfdx_cmd += " " + shell_quote(arg)
     if username:
-        command += f" -u {shell_quote(username)}"
+        sfdx_cmd += f" -u {shell_quote(username)}"
     if log_note:
-        logger.info(f"{log_note} with command: {command}")
+        logger.info(f"{log_note} with command: sfdx {command}")
+        plog_command_options(args)
```

Comment:
Your indentation has this beneath the conditional, so it will only run when log_note is passed. We're only doing this in a handful of places, so it won't run for most commands. Is this what you want?